test/test_torch.py

```diff
@@ -16140,17 +16152,26 @@ def test_addmm_sizes(self, device, dtype):
         for m in [0, 1, 25]:
             for n in [0, 1, 10]:
                 for k in [0, 1, 8]:
+                    if False and dtype.is_complex:  # bug to be fixed in another PR
+                        alpha = 1.2 + 0.3j
+                        beta = 0.5 + 1.6j
+                    else:
+                        alpha = 1.2
+                        beta = 0.8
                     M = torch.randn(n, m, device=device, dtype=dtype)
                     m1 = torch.randn(n, k, device=device, dtype=dtype)
                     m2 = torch.randn(k, m, device=device, dtype=dtype)
-                    res1 = torch.addmm(M, m1, m2)
-                    res2 = torch.zeros(n, m, device=device, dtype=dtype)
-                    res2 += M
+                    res1 = torch.addmm(M, m1, m2, alpha=alpha, beta=beta)
+                    res2 = torch.full_like(res1, math.nan)
+                    torch.addmm(M, m1, m2, alpha=alpha, beta=beta, out=res2)
+                    res3 = (beta * M).cpu().tolist()
                     for i in range(n):
                         for j in range(m):
                             for l in range(k):
-                                res2[i, j] += m1[i, l] * m2[l, j]
+                                res3[i][j] += (alpha * m1[i, l] * m2[l, j]).item()
```

Comment:
`.item` can be removed with https://github.com/pytorch/pytorch/pull/43844

Reply:
#43844 is merged. Also, comparison with numpy reference here.

Reply:
fixed